app/src/main/java/org/wildaid/ofish/ui/crew/CrewFragment.kt

```diff
@@ -97,7 +97,7 @@ class CrewFragment : BaseReportFragment(R.layout.fragment_crew) {
         hideKeyboard()
         when (buttonId) {
             R.id.btn_next -> {
-                if (isFieldCheckPassed || isAllRequiredFieldsNotEmpty()) {
+                if (isFieldCheckPassed || validateForms()) {
```

Comment:
The method is changed. Do we need to check both conditions?

Reply:
This is just renaming of method, since it has side effect under the hood. 
Boolean value indicates the user has already passed validation. It prevents preventing user to navigate futher